deployment_report/model/static/viya_deployment_report_keys.py

```diff
@@ -78,3 +78,15 @@ class Relationship(object):
                 """
                 RESOURCE_NAME = "resourceName"
                 RESOURCE_TYPE = "resourceType"
+
+    class DatabaseDetails(object):
+        """
+        Class defining static references to keys in Database details dictionaries in resulting
+        ViyaDeploymentReport data structure.
+        """
+        DBHOST = "Host"
+        DBNAME = "Name"
+        DBPORT = "Port"
+        DBTYPE = "Type"
+        DBSSL = "SSL_Enabled"
+        DBCONN = "Connection"
```

Comment:
Moving the keys into this file was a good idea. Thank you for doing so.

Unfortunately, the way they're defined here still yields the same issue as before. If you take a look at the other keys in this file, everything is in lower camel-case (e.g., "resourceType") and none of the keys contain an underscore (other than a few exceptions that start with an underscore to represent metadata about the report). The DatabaseDetails keys are mostly title-cased (i.e., first letter capitalized). So in the resulting json most keys will be defined like:

```json
{
    "resourceName": "",
    "resourceType": ""
}
```
However, these keys will be defined like:

```json
{
    "Host": "",
    "Name": "",
    "Port": ""
}
```
So this would introduce an inconsistency in the data definitions. These keys should be formatted in lower camel-case to match the rest of the keys. If they need to displayed in title case in the HTML report, then jinja2 provides a filter called "title" to capitalize the first letter of the word: `{{ key | title }}`

This sort of formatting for display should be done in the jinja2 template instead of in the data itself. If keys do not title-case nicely, then it may just be worth hard-coding the table row header and then reference the expected data:

```
<table>
  <tr>
    <th>Host</th>
    <td>{{ info.host }}</td>
  </tr>
  <tr>
    <th>SSL Enabled</th>
    <td>{{ info.ssl }}</td>
  </tr>
  <!- continued for all rows -->
</table>
```